assets/queries/dockerfile/unpinned_package_version_in_pip_install/query.rego

```diff
@@ -1,21 +1,19 @@
 package Cx
 
 CxPolicy[result] {
-	from := input.document[i].command[j]
-	instruction := from[k]
+	resource := input.document[i].command[name][_]
```

Comment:
don't use the name resource here, because in dockerfile it's just commands/instructions being run